### DISCUSSION TITLE
Your favorite country or places you would love to visit in the next 5 years ?

### DISCUSSION THREAD

**Discussion by Samabena** [NAIST-SE/AttractingContributors#1071] [2024 Attraction Contributions]
We all dream of visiting some places or countries in the near future. 
IMAGINE you have the MONEY and the OPPORTUNITIES to travel ANYWHERE you want without any RESTRICTIONS. 
Where would you love to go, and what places would you love to visit?

NB : Your DREAM will COME TRUE if you write it down, quickly [click here](https://github.com/Samabena/sighting/blob/main/README.md) to write it down and observe what happens in the near future. 

Thanks in advance for your contributions.

**Comment by Drobotsan** [comment]
Hi, my topic is similar to yours

Would you be kind to contribute with my project :D
https://github.com/Drobotsan/Places-in-the-world

**Comment by thapakorn604** [comment]
Hi, I have created pull request to your topic -> https://github.com/Samabena/sighting/pull/15
Would you mind to make one to mine ? -> https://github.com/NAIST-SE/AttractingContributors/discussions/1115

**Comment by 3207-Rhims** [comment]
Hi, Would you be kind to be contribute in my project. 

List of movie #1164

**Comment by Marsall-k** [comment]
I have just made a PR on your Repo!
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon

**Comment by Idescrew** [comment]
I have just made a PR on your Repo!
Could you contribute to mine?

https://github.com/Idescrew/AttractingContributors
